<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="dropShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame {
	background-color: rgb(255, 57, 57);
	color: rgb(255, 220, 221);
	border-radius: 12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>661</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 211, 235);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Youtube&lt;/strong&gt; to &lt;strong&gt;MP3/MP4&lt;/strong&gt; </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_description">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 143, 145);</string>
       </property>
       <property name="text">
        <string>Easily translate &lt;strong&gt;Youtube&lt;/strong&gt; videos to &lt;strong&gt;mp3/mp4&lt;/strong&gt; files!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>280</y>
         <width>501</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar{
	background-color: rgb(255, 165, 167);
	color: rgb(255, 255, 255);
	border-style: none;
	border-radius: 12px;
	text-align: center;
}

QProgressBar::chunk{
	border-radius: 12px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.557, x2:1, y2:0.557, stop:0 rgba(170, 55, 56, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>661</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 143, 145);</string>
       </property>
       <property name="text">
        <string>Loading...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>350</y>
         <width>611</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 143, 145);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Created&lt;/strong&gt; by Rafael Malcervelli</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
